index.html

```diff
@@ -1073,18 +1073,6 @@ <h2>You got to the end!</h2>
         correct: 0,
         explanation: 'toTimeString() returns the string "Invalid Date" for invalid dates. 🫠'
       },
-
-      {
-        code: 'new Date("1")',
-        options: [
-          '1970-01-01T00:00:01.000Z',
-          '2001-01-01T00:00:00.000Z',
-          '0001-01-01T00:00:00.000Z',
-          'Invalid Date'
-        ],
-        correct: 1,
-        explanation: 'Just like "0", "1" is interpreted as a year, so this is 2000+1.'
-      },
```

Comment:
Could we keep this in, please? 🙏 I don't want to change the number of questions.

Reply:
I think it's quite weird to have:
"0" year
"1" month
"2" also month
"12" also month
"13" also month but invalid

Maybe there's another weird example that we can replace it with, that would probably make the most sense.

Reply:
Yeah, I think if I could wind back time I'd remove _one_ of those (probably 12), but the problem with doing it now is that so many people have already done the quiz and have shared their score out of 28, and remember the questions that caused them the most trauma. I think I would prefer to keep the quiz the same so people know they're comparing against exactly what their friends/colleagues did.